### PR TITLE
WA-432 Update UI on boarding screens

### PR DESCRIPTION
**Pull request by germartinez**
New user interface for the browser extension.
Onboarding process styles and new screen flow.

Welcome page:
welcome

Accept terms and conditions:
terms_conditions

Create the master password:
create_password

Master password must meet some requirements:
create_password_error
create_password_ok

Confirm the master password:
confirm_password

Confirmed password must be equal to the master password:
confirm_password_error
confirm_password_ok

Next steps: download the app and do the pairing
downloads

qr-code to download the ios app:
ios_download

qr-code to download the android app:
android_download

Pairing qr-code:
pairing

Unlocking the ethereum account in the extension:
unlocking

### CODE REVIEW

**Comment by apanizo** [comment]
This PR is huge to review, I know there is no easy way to have a deployed version to test it, but it would be really helpful if you add: 
* BEFORE && AFTER screenshots
* Image pointing which components are new and which views should be reviewed

Can you complete the info, please?

File: app/routes/ConfirmPassword/components/Layout.js [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
+import classNames from 'classnames/bind'
 
 import Page from 'components/Page'
 import ConfirmPasswordForm from './ConfirmPasswordForm/containers/ConfirmPasswordForm'
-import styles from './index.css'
+import styles from 'assets/css/global.css'
+
+const cx = classNames.bind(styles)
 
 class Layout extends Component {
```
Comment (germartinez) [inline app/routes/ConfirmPassword/components/Layout.js]: confirm_password
confirm_password_error
confirm_password_ok

File: app/routes/CreatePassword/components/Layout.js [hunk 2]
```diff
@@ -1,9 +1,12 @@
...
+import classNames from 'classnames/bind'
 
 import Page from 'components/Page'
 import CreatePasswordForm from './CreatePasswordForm/containers/CreatePasswordForm'
-import styles from './index.css'
+import styles from 'assets/css/global.css'
+
+const cx = classNames.bind(styles)
 
 class Layout extends Component {
```
Comment (germartinez) [inline app/routes/CreatePassword/components/Layout.js]: create_password
create_password_error
create_password_ok

File: app/routes/DownloadApps/components/AppQr/AppQr.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react'
+import { Link } from 'react-router-dom'
+
+import { createQrImage } from 'utils/qrdisplay'
+import styles from 'assets/css/global.css'
+
+class AppQr extends Component {
```
Comment (germartinez) [inline app/routes/DownloadApps/components/AppQr/AppQr.js]: ios_download
android_download

File: app/routes/DownloadApps/components/PairingProcess/components/Layout.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react'
+
+import CreateAccount from './CreateAccount'
+import LockedAccount from './LockedAccount'
+import UnlockedAccount from './UnlockedAccount'
+import styles from 'assets/css/global.css'
+
+class Layout extends Component {
```
Comment (germartinez) [inline app/routes/DownloadApps/components/PairingProcess/components/Layout.js]: pairing

File: app/routes/DownloadApps/containers/DownloadApps.js [hunk 5]
```diff
@@ -1,40 +1,60 @@
 import React, { Component } from 'react'
 
 import Layout from '../components/Layout'
-import { createQrImage } from 'utils/qrdisplay'
 import config from '../../../../config'
 
 class DownloadApps extends Component {
```
Comment (germartinez) [inline app/routes/DownloadApps/containers/DownloadApps.js]: downloads

File: app/routes/Password/containers/Password.js [hunk 6]
```diff
@@ -13,38 +13,51 @@ class Password extends Component {
...
     }
-    this.setState({ errorMessage: 'Wrong password' })
+
+    this.setState({ dataValidation: 'ERROR' })
+    setTimeout(() => {
+      this.setState({ dataValidation: '' })
+    }, 500)
   }
 
   render() {
```
Comment (germartinez) [inline app/routes/Password/containers/Password.js]: unlocking

File: app/routes/Welcome/containers/Welcome.js [hunk 7]
```diff
@@ -1,10 +1,31 @@
...
 import Layout from '../components/Layout'
 
-const Welcome = () => (
-  <Layout />
-)
+class Welcome extends Component {
+  constructor(props) {
+    super(props)
 
-export default Welcome
```
Comment (germartinez) [inline app/routes/Welcome/containers/Welcome.js]: welcome
terms_conditions

**Pull request merged**